week2/homework/src/index.js

```diff
@@ -1,3 +1,134 @@
 'use strict';
+const fs = require('fs');
+const command = process.argv[2];
+const commandFirstItem = process.argv[3];
+const newData = process.argv[4];
 
-// TODO: Write the homework code in this file
+function help() {
+  fs.readFile('./help.txt', 'utf8', function (error, data) {
+    if (error) {
+      console.log(error);
+    }
+    else {
+      console.log(data);
+    }
+  });
+};
+
+function list() {
+  fs.readFile('./toDo.json', 'utf8', function (error, data) {
+    if (error) {
+      console.log(error);
+    }
+    else {
+      const myToDoList = JSON.parse(data);
+      if (Object.values(myToDoList).length === 0) {
+        console.log('You have not any plan in your To Do List');
+      }
+      else {
+        console.log(myToDoList);
+      }
+    }
+  });
+};
+
+function add(commandFirstItem) {
+  fs.readFile('./toDo.json', 'utf8', function (error, data) {
+    if (error) {
+      console.log(error);
+    }
+    else {
+      const myToDoList = JSON.parse(data);
+      const index = Object.keys(myToDoList).length + 1;
+      myToDoList[index] = commandFirstItem;
```

Comment:
This works as intended, but you are using an object as an array.
There's nothing wrong with it in this case as it does what you want, but it's not the javascript way.

Try to rewrite the app so that the json looks like this:
```js
[
  {
    index: 1,
    description: 'Swim lesson 12.10.18 15:00 - 16:00'
  },
  {
    index: 2,
    description: 'Baseball lesson 13.10.18 15:00 - 16:00'
  },
];
```

So you have an array of objects to work with.

Reply:
After editing, Now program works on arrays